js/load.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+(() => {
+  const URL = 'https://21.javascript.pages.academy/keksobooking/data';
+  const TIMEOUT_IN_MS = 2000;
+  const StatusCode = {
+    OK: 200
+  };
+
+  window.load = (onSucces, onError) => {
+    const xhr = new XMLHttpRequest();
+    xhr.responseType = 'json';
+
+    xhr.addEventListener('load', () => {
+      if (xhr.status === StatusCode.OK) {
+        onSucces(xhr.response);
+      } else {
+        onError(`Response status: ${xhr.status} ${xhr.statusText}`);
+      }
+    });
+    xhr.addEventListener(`error`, () => {
+      onError(`Error connection`);
+    });
+    xhr.addEventListener('timeout', () => {
+      onError(`Error of connection after ${xhr.timeout} мс`);
```

Comment:
1. Тут у тебя смешивается русский и англ языки
2. Страница на русском (lang=ru), ошибки также стоит выводить на русском и сообщать код ошибки. Так пользователю будет легче обращаться в службу поддержки
```suggestion
      onError(`Запрос не успел выполниться за ${xhr.timeout}мс. Статус ответа: ${xhr.status} ${xhr.statusText}. Попробуйте перезагрузить страницу`);
```